src/agent/Core/ApplicationPool/Group.h

```diff
@@ -202,6 +200,8 @@ class Group: public boost::enable_shared_from_this<Group> {
 	Callback shutdownCallback;
 	GroupPtr selfPointer;
 
+	// Whether to use the old routing algorithm
+	bool oldRouting;
```

Comment:
Since on the web server level this is a global config anyway, you can turn this into a pool-level flag (i.e., a boolean in ApplicationPool2::Context) instead of a Group-level flag. That simplifies some code.

Reply:
Done